pkg/container/docker_run.go

```diff
@@ -276,8 +275,11 @@ func (cr *containerReference) connect() common.Executor {
 func (cr *containerReference) Close() common.Executor {
 	return func(ctx context.Context) error {
 		if cr.cli != nil {
-			cr.cli.Close()
+			err := cr.cli.Close()
 			cr.cli = nil
+			if err != nil {
+				return fmt.Errorf("failed to close client: %w", err)
```

Comment:
Maybe this is more likely a logged warning? Why fail completly in this case?

Also you do ignore the errors over here: https://github.com/nektos/act/pull/1077/files#diff-ac725a1756aff41010f7e72b3e3f0f63677898bd5eea991096438ffe9c45f9ec